easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2b-GCC-4.9.3-2.25.eb

```diff
@@ -12,6 +12,12 @@ sources = [SOURCELOWER_TAR_GZ]
 # Let's store the checksum in order to be sure it doesn't suddenly change
 checksums = ['5651e8b7a72d7c77ca68da48f3a5d108']
 
-builddependencies = [('Bison', '3.0.4')]
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('libibmad', '1.3.12'),
+    ('libibumad', '1.3.10.2'),
```

Comment:
@boegel - that's a good point, that I need to check.

In my testing, using the OS deps didn't work because the headers weren't available during the `configure` step, which is why I did it this way.

Is there a way of specifying build deps as using OS packages?